MapleServer2/PacketHandlers/Game/FieldEnterHandler.cs

```diff
@@ -23,31 +23,34 @@ public override void Handle(GameSession session, PacketReader packet)
         Player player = session.Player;
         Account account = player.Account;
         session.EnterField(player);
-        session.Send(StatPacket.SetStats(session.Player.FieldPlayer));
+        session.Send(StatPacket.SetStats(player.FieldPlayer));
         session.Send(StatPointPacket.WriteTotalStatPoints(player));
 
         if (account.IsVip())
         {
-            session.Send(BuffPacket.SendBuff(0, new(100000014, session.Player.FieldPlayer.ObjectId, session.Player.FieldPlayer.ObjectId, 1, (int) account.VIPExpiration, 1)));
-            session.Send(PremiumClubPacket.ActivatePremium(session.Player.FieldPlayer, account.VIPExpiration));
+            session.Send(BuffPacket.SendBuff(0,
+                new(100000014, player.FieldPlayer.ObjectId, player.FieldPlayer.ObjectId, 1, (int) account.VIPExpiration, 1)));
```

Comment:
Maybe doesn't have to be in this PR but the buff isn't 100000014 in GMS. It's 100000043 and 100000045. These should be parsed from /table/vipbenefiteffecttable.xml

Reply:
Lets do it in another PR!